INSTALL.md

```diff
@@ -1,55 +1,36 @@
 
-    Examples: `/home/myuser/www/web/` or `/var/sites/ezpublish/web/`
+    Examples: `/home/myuser/www/web/` or `/var/sites/<project-name>/web/`
 
-## Install all components
+## Clean install
 
-1. You can get eZ Publish using GIT with the following command:
+1. You can get eZ Platform using composer with the following commands:
 
        ```bash
-       git clone https://github.com/ezsystems/ezpublish-community.git
-       ```
-
-2. *Optional* Upgrade an eZ Publish installation
-
-       Follow normal eZ Publish upgrade procedures for upgrading the distribution files and moving over extensions as found for instance here:
-       http://doc.ez.no/eZ-Publish/Upgrading/Upgrading-to-5.0/Upgrading-from-4.7-to-5.0
-
-3. Install the dependencies with [Composer](http://getcomposer.org).
-
-       **Note: The following step will also install assets, if you prefer to install assets using hard copy or symlink
-               instead of default relative symlink, edit 'symfony-assets-install' setting in composer.json**
-
-       **Dev: For dev use change '--prefer-dist' for '--prefer-source' to get full git clones
-              and add '--dev' to get phpunit and behat installed.**
-
-       Download composer and install dependencies by running:
-       ```bash
-       cd /<ezpublish-community-root-dir>/
        curl -s http://getcomposer.org/installer | php
-       php -d memory_limit=-1 composer.phar install --prefer-dist
+       php -d memory_limit=-1 composer.phar create-project --prefer-dist ezsystems/ezpublish-community <project-name> <version>
+       cd /<project-name>/
        ```
+       
+      Options:
+      - `<version>`: `dev-master` to get current development version (pre release), `v2015.01` to pick a specific release, otherwise skip it to get latest stable release.
+      - For core development change '--prefer-dist' to '--prefer-source' to get full git clones, and add '--dev' to get phpunit and behat installed.
 
-       Update note: Every time you want to get the latest updates of all your dependencies just run this command:
-       ```bash
-       cd /<ezpublish-community-root-dir>/
-       php -d memory_limit=-1 composer.phar update --prefer-dist
-       ```
+  After running command above, composer will take it's time to download all libraries, then a wizard will ask you for setting to database and lastly a welcome screen should grant you instructions for finishing installation.
 
-4. Setup folder rights **For *NIX users**:
 
-       One common issue is that the `ezpublish/cache`, `ezpublish/logs` and `ezpublish/config` directories **must be writable both by the web server and the command line user**.
+2. Setup folder rights **For *NIX users**:
+
+       One common issue is that the `ezplatform/cache`, `ezplatform/logs` and `ezplatform/config` directories **must be writable both by the web server and the command line user**.
```

Comment:
These renames can not be done unless we rename the folder.

@lolautruche Any POV on renaming this folder? We talked about using app at some point so Symfony doc is correct, could also be ezp. Anyway, I think I'll change these doc back for now.